storybookjs/storybook
# fix: storybook & webpack 5 successfully build

Issue: Build didn't work 

## What I did
- updated fork ts plugin
- updated webpack to v5 stable
- implemented mjs bandaid loader
- updated loader options API
- kind of fixed cache issues, but needs improvement
- updated webpack virtual modules
- fixed incorrect devtools
## How to test

i ran one example and it seems to build. needs more testing from those who know storybook 

If your answer is yes to any of these, please make sure to include it in your PR.

```
Snapshot Summary
 › 5 snapshots failed from 2 test suites. Inspect your code changes or re-run jest with `-u` to update them.

Test Suites: 5 failed, 121 passed, 126 total
Tests:       9 failed, 1452 passed, 1461 total
Snapshots:   5 failed, 416 passed, 421 total
Time:        75.798 s
Ran all test suites in 7 projects.
```



**there are many references to webpack 4 in here. If I do a global search, everything needs to updated to webpack 5. You can use syncpack to sync it every**
